src/js/components/HeaderBar.tsx

```diff
@@ -9,10 +10,12 @@ export default function HeaderBar() {
 
   return (
     <UIHeaderBar>
-      <span>toggle component here</span>
-      <span>logo component here</span>
-      <span>user-menu component here</span>
-      <span>cluster-menu component here</span>
+      {/* <span>toggle component here</span> */}
```

Comment:
please do not touch these lines to prevent conflicts 👍

Reply:
Good point